src/Symfony/Component/Security/Core/Authorization/ExpressionLanguage.php

```diff
@@ -13,6 +13,10 @@
 
 use Symfony\Component\ExpressionLanguage\ExpressionLanguage as BaseExpressionLanguage;
 
+if (!class_exists('Symfony\Component\ExpressionLanguage\ExpressionLanguage')) {
+    throw new \LogicException('Install symfony/expression-language');
```

Comment:
To be consistent with (the most) similar exceptions already in the code, what about throwing a \RuntimeException with a more precise message like "The ExpressionLanguage class requires the "ExpressionLanguage" component. Install "symfony/expression-language" to use it." ?

Reply:
Ok, thanks!

Reply:
Done